FogCreek/Glitch-Community
# Mirror spell: refactor team name & url inputs

## Links
* Remix link: https://mirror-spell.glitch.me

## Changes:
* inline team-name-input and team-url input into team-fields
* extract non-editable team name & input from team page
* componentize VerifiedBadge

## Why does this exist
A lot of this work was done in a separate PR that went off the rails (#492); this is a self-contained chunk that doesn't need to be reviewed as part of that PR.

## How To Test:
* visit a glitch team you are not an admin of, e.g. https://mirror-spell.glitch.me/@community
* this team should have non-editable fields for name and url
* visit or create a glitch team which you have admin privileges
* the team should have editable fields
* changing those fields should update the team name or URL
* verified teams should have the checkmark, e.g. https://mirror-spell.glitch.me/@slack
